pipeline_steps/deploy.groovy

```diff
@@ -36,7 +36,7 @@ def deploy_sh(Map args) {
             }
             sh """#!/bin/bash
             sudo ssh -T -oStrictHostKeyChecking=no ${args.vm} \
-              '${export_vars} cd /opt/rpc-openstack; scripts/deploy.sh'
+              '${export_vars} cd /opt/rpc-openstack; FORKS=5; scripts/deploy.sh'
```

Comment:
hi pls ignoring it, because the vm only has 4 cores and i want to change the ansible fork number to 5 for stability. i should closed this ticket first then reopen it